zerver/openapi/zulip.yaml

```diff
@@ -1135,6 +1135,241 @@ paths:
+                          {
+                             "date_joined": "2019-10-20T07:50:53.729659+00:00",
+                             "full_name": "King Hamlet",
+                             "is_guest": false,
+                             "profile_data": {
+                                "4": {
+                                   "value": "vim"
+                                },
+                                "2": {
+                                   "value": "I am:\n* The prince of Denmark\n* Nephew to the usurping Claudius",
+                                   "rendered_value": "<p>I am:</p>\n<ul>\n<li>The prince of Denmark</li>\n<li>Nephew to the usurping Claudius</li>\n</ul>"
+                                },
+                                "5": {
+                                   "value": "1900-1-1"
+                                },
+                                "7": {
+                                   "value": "[11]"
+                                },
+                                "6": {
+                                   "value": "https://blog.zulig.org"
+                                },
+                                "1": {
+                                   "value": "+0-11-23-456-7890",
+                                   "rendered_value": "<p>+0-11-23-456-7890</p>"
+                                },
+                                "8": {
+                                   "value": "zulipbot"
+                                },
+                                "3": {
+                                   "rendered_value": "<p>Dark chocolate</p>",
+                                   "value": "Dark chocolate"
+                                }
+                             },
+                             "user_id": 10,
+                             "is_bot": false,
+                             "bot_type": null,
+                             "timezone": "",
+                             "is_admin": false,
+                             "avatar_url": "https://secure.gravatar.com/avatar/6d8cad0fd00256e7b40691d27ddfd466?d=identicon&version=1",
+                             "is_active": true,
+                             "email": "hamlet@zulip.com"
+                          }
+                       ]
+                    }
+
+    patch:
+      description: Update the user.
+      parameters:
+      - name: user_id
+        in: path
+        description: The user id of the user whose data you want to update.
+        schema:
+          type: integer
+        example: 12
+        required: true
+      - name: full_name
+        in: query
+        description: The full name of the user will be updated with the new full_name passed.
+        schema:
+          type: string
+          format: json
+        example: NewName
+        required: false
+      - name: is_admin
+        in: query
+        description: The admin status of the user will be updated to the new 'is_admin' boolean passed.
```

Comment:
This is awkward language; how about just explaining what the field means, e.g. "Whether the target user is an administrator|guest" for these.

Or "The user's full name"